tests/integration/targets/test_mysql_user/tasks/main.yml

```diff
@@ -281,6 +281,10 @@
     - include: test_priv_subtract.yml enable_check_mode=no
     - include: test_priv_subtract.yml enable_check_mode=yes
 
+    - include: test_privs_issue_465.yml
+      tags:
```

Comment:
This should not work. For an import it is ok, but not for include. I tested, it work and I've no idea why. Normally I would have added:

```yml
apply:
  tags:
    - issue_465
```
But I prefer not to touch this as I never use the short form of include and thus can't be sure.
Also, for this, I think an import was better.